avm/res/db-for-postgre-sql/flexible-server/tests/e2e/defaults/main.test.bicep

```diff
@@ -35,21 +31,43 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2023-07-01' = {
   location: resourceLocation
 }
 
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
+  params: {
+    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
+    location: resourceLocation
+  }
+}
 // ============== //
 // Test Execution //
 // ============== //
 
 @batchSize(1)
-module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
-  scope: resourceGroup
-  name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
-  params: {
-    name: '${namePrefix}${serviceShort}001'
-    location: resourceLocation
-    administratorLogin: 'adminUserName'
-    administratorLoginPassword: password
-    skuName: 'Standard_B2s'
-    tier: 'Burstable'
-    geoRedundantBackup: 'Enabled'
+module testDeployment '../../../main.bicep' = [
+  for iteration in ['init', 'idem']: {
+    scope: resourceGroup
+    name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
+    params: {
+      name: '${namePrefix}${serviceShort}001'
+      location: resourceLocation
+      administrators: [
+        {
+          objectId: nestedDependencies.outputs.managedIdentityClientId
+          principalName: nestedDependencies.outputs.managedIdentityName
+          principalType: 'ServicePrincipal'
+        }
+      ]
+      skuName: 'Standard_D2s_v3'
+      tier: 'GeneralPurpose'
+      geoRedundantBackup: 'Enabled'
+      maintenanceWindow: {
```

Comment:
Should not be here, as not a required value. 
To satisfy the WAF rule, this parameter should either have a default value in the `main.bicep` - or - we should supress it in the PSRule settings for the defaults test.
Same for the other parameters that were added.

If user input is required its the WAF test that should satisfy the rules while the defaults test depends on the defaults or exceptions